bin/registerServerTopology.php

```diff
@@ -42,28 +42,28 @@
 $helpMessage = <<<'EOD'
```

Comment:
It would be useful if you describe the general purpose of the script before talking about the options ?